Packs/CrowdStrikeFalcon/ReleaseNotes/2_0_26.md

```diff
@@ -0,0 +1,21 @@
+
+#### Playbooks
+
+##### CrowdStrike Falcon - False Positive Incident Handling
+
+- Updated the playbook to be availble only for XSOAR marketplace.
+##### CrowdStrike Falcon - Search Endpoints By Hash
+
+- Updated the playbook to be availble only for XSOAR marketplace.
+##### CrowdStrike Falcon Malware - Investigation and Response
+
+- Updated the playbook to be availble only for XSOAR marketplace.
+##### CrowdStrike Falcon - True Positive Incident Handling
+
+- Updated the playbook to be availble only for XSOAR marketplace.
+##### CrowdStrike Falcon - SIEM ingestion Get Incident Data
+
+- Updated the playbook to be availble only for XSOAR marketplace.
```

Comment:
based on my last comment - this might need an update

Reply:
NP make sense